frontend/src/views/DashboardHeader.vue

```diff
@@ -40,6 +40,12 @@
             class="bar-link px-2 py-2 rounded-md"
             >{{ $t("common.settings") }}</router-link
           >
+          <router-link
+            v-if="isDevFeatures"
```

Comment:
I see we have already commented out the existing console code, then let's just enable this for all env (yeah, let's try to get this to a MVP launch stage next week)